README.md

```diff
@@ -294,6 +299,27 @@ delta a.txt b.txt
 diff -u a.txt b.txt | delta
 ```
 
+#### Environment
+
+##### a) Delta
+Delta respects the setting of some environment variables to derive the default pager to use. These are in descending order of priority :
+- `DELTA_PAGER`,
+- `BAT_PAGER` and
+- `PAGER`
+
+Hence if e.g. `$DELTA_PAGER` is unset, delta will use `$BAT_PAGER`, or `$PAGER` if that is unset, too. In environments wthout any of the three variable set, delta's fallback is `less`.
+
+_**NOTE:** Be aware that setting one of those environment variables to `delta` while leaving the ones with higher priority unset (or also pointing at `delta`) can lead to an infinite cycle of forked processes by delta recursively calling itself._
+
```

Comment:
OK I hope this doesn't sound like I'm being difficult :)  but I don't think we need to mention this in the README as it's an unlikely scenario and doesn't correspond to any recommended usage. If it turns out to be a problem that users encounter in practice, then we should just put a fix in the code so that delta refuses to call itself.

Reply:
NP :smile: